types/lodash/common/string.d.ts

```diff
@@ -29,19 +29,19 @@ declare module "../index" {
          * @param string The string to capitalize.
          * @return Returns the capitalized string.
          */
-        capitalize(string?: string): string;
+        capitalize<T extends string>(string?: T): Capitalize<T>;
```

Comment:
Lodash capitalize not only capitalizes the first letter, it also concerts the rest to lowercase https://lodash.com/docs#capitalize. So I think the return type should be:

`Capitalize<Lowercase<T>>`

Reply:
Oh, I missed that. Thanks!